### PR TITLE
Added RN update regarding route with externally managed certificate

### PR DESCRIPTION
**Pull request by subhtk**
Version(s): 4.16

Issue: https://issues.redhat.com/browse/OCPBUGS-36310

Link to docs preview: https://78291--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-networking-secured-route_release-notes

https://78291--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-technology-preview-tables_release-notes

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by lihongan** [comment]
LGTM
Thank you @subhtk

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -833,6 +833,12 @@ This release introduces `FRR-K8s`, a Kubernetes based `DaemonSet` that exposes a
 
 For more information, see xref:../networking/metallb/metallb-frr-k8s.adoc#metallb-configure-frr-k8s[Configuring the integration of MetalLB and FRR-K8s].
 
+[id="ocp-4-16-networking-secured-route_{context}"]
+==== Creating a route with externally managed certificate (Technology Preview)
+
+
+To configure {product-title} routes with third-party certificate management solutions, utilize the `.spec.tls.externalCertificate` field in the route API. This allows you to reference externally managed TLS certificates through secrets, streamlining the process by eliminating manual certificate management. By using externally managed certificates, you reduce errors, ensure a smoother certificate update process, and enable the OpenShift router to promptly serve renewed certificates. For more information, see xref:https:../networking/routes/secured-routes.html#nw-ingress-route-secret-load-external-cert_secured-routes[Creating a route with externally managed certificate].
```
Comment (chiragkyal) [inline release_notes/ocp-4-16-release-notes.adoc:840]: Can we write the phrase `With this release` to resemble the existing RNs. ?

```suggestion
With this release, {product-title} routes can be configured with third-party certificate management solutions, utilising the `.spec.tls.externalCertificate` field in the route API. This allows you to reference externally managed TLS certificates through secrets, streamlining the process by eliminating manual certificate management. By using externally managed certificates, you reduce errors, ensure a smoother certificate update process, and enable the OpenShift router to promptly serve renewed certificates. For more information, see xref:https:../networking/routes/secured-routes.html#nw-ingress-route-secret-load-external-cert_secured-routes[Creating a route with externally managed certificate].
```

**Review by JoeAldinger — COMMENTED** [review]
/lgtm

**Comment by JoeAldinger** [comment]
Merge requested via slack.

**Pull request merged**